addons/point_of_sale/static/src/app/printer/base_printer.js

```diff
@@ -28,7 +27,8 @@ export class BasePrinter {
         let image, printResult;
         while (this.receiptQueue.length > 0) {
             receipt = this.receiptQueue.shift();
-            image = await this.htmlToImg(receipt);
+            image = this.processCanvas(await toCanvas(receipt, "pos-receipt-print"));
+            receipt.remove();
```

Comment:
this is different `receipt` from the original `htmlToImg`. Can't call `remove` method here.